state/applicationoffers_test.go

```diff
@@ -167,6 +167,22 @@ func (s *applicationOffersSuite) TestAddApplicationOfferBadEndpoints(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 }
 
+func (s *applicationOffersSuite) TestAddApplicationOfferContainerScopedEndpoints(c *gc.C) {
+	s.AddTestingApplication(c, "local-wordpress", s.AddTestingCharm(c, "wordpress"))
+	// logging-dir is a container scoped relation.
+	eps := map[string]string{"logging-dir": "logging-dir"}
+	sd := state.NewApplicationOffers(s.State)
+	owner := s.Factory.MakeUser(c, nil)
+	args := crossmodel.AddApplicationOfferArgs{
+		OfferName:       "offer-name",
+		ApplicationName: "local-wordpress",
+		Endpoints:       eps,
+		Owner:           owner.Name(),
+	}
+	_, err := sd.AddOffer(args)
+	c.Assert(err, gc.ErrorMatches, `.*scope "container".*`)
```

Comment:
```suggestion
	c.Assert(err, gc.ErrorMatches, `.*can only offer endpoints with global scope, provided scope "container".*`)
```

Reply:
Changed.

I wasn't sure about weather to match the error check to the exact error message in case of change of wording, but it does seem better to just simply change the test if you change the wording since its an error from the file we're testing.